File: PromoStudio.Common/Models/AudioScriptTemplate.cs
using System;
using PromoStudio.Common.Enumerations;

namespace PromoStudio.Common.Models
{
    public class AudioScriptTemplate
    {
        public int pk_AudioScriptTemplateId { get; set; }
        public sbyte fk_AudioScriptTemplateStatusId { get; set; }
        public int? fk_OrganizationId { get; set; }
        public int? fk_VerticalId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string ScriptText { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime? DateUpdated { get; set; }

        public AudioScriptTemplateStatus Status
        {
            get { return (AudioScriptTemplateStatus) fk_AudioScriptTemplateStatusId; }
            set { fk_AudioScriptTemplateStatusId = (sbyte) value; }
        }

        public dynamic ToPoco()
        {
            return new
            {
                pk_AudioScriptTemplateId,
                fk_AudioScriptTemplateStatusId,
                fk_OrganizationId,
                fk_VerticalId,
                Name,
                Description,
                ScriptText,
                DateCreated,
                DateUpdated
            };
        }
    }
}

File: PromoStudio.Web/Scripts/models/audioScriptTemplate.js
/// <reference path="../vsdoc/require.js" />
/// <reference path="../vsdoc/knockout-3.0.0.debug.js" />

"use strict";

define(["knockout"], function (ko) {
    function ctor (data) {

        var self = this;
        data = data || {};

        self.pk_AudioScriptTemplateId = ko.observable(data.pk_AudioScriptTemplateId || null);
        self.fk_AudioScriptTemplateStatusId = ko.observable(data.fk_AudioScriptTemplateStatusId || null);
        self.fk_OrganizationId = ko.observable(data.fk_OrganizationId || null);
        self.fk_VerticalId = ko.observable(data.fk_VerticalId || null);
        self.Name = ko.observable(data.Name || null);
        self.Description = ko.observable(data.Description || null);
        self.ScriptText = ko.observable(data.ScriptText || null);
        self.DateCreated = ko.observable(data.DateCreated || null);
        self.DateUpdated = ko.observable(data.DateUpdated || null);
    }

    ctor.prototype.toJSON = function () {
        var copy = ko.toJS(this);
        // remove any unneeded properties
        delete copy.DateCreated;
        delete copy.DateUpdated;

        return copy;
    };
    return ctor;
});